src/content/app-platform/app-bundle/index.md

```diff
@@ -0,0 +1,238 @@
+
+<h5><i class="fa fa-help-outline"></i> How to best make the names unique?</h5> It is a common practice to either prepend or append the Workload Cluster name to the App CR name, creating for example "wc1-bundle" or "bundle-wc1".
+
+</div>
+
+![User installs a bundle twice with correct names](app-bundle-naming-conflict-step-3.png)
+
+#### Configuring the bundle's App CR for the unique App Operator
+
+Once unique names are ensured, the App CR must be configured to be installed in the Management Cluster by the
+unique App Operator. It is done by the means of the `app-operator.giantswarm.io/version: 0.0.0` label and
+the `.spec.kubeConfig.inCluster: true` field. See the snippet below, note some fields have been removed for
+brevity.
+
+```yaml
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+  name: bundle-wc1
+  ...
+spec:
+  ...
+  kubeConfig:
+    inCluster: true
+  name: bundle-app
+  ...
+```
+
+These two settings ensure this App CR is picked up and reconciled by the unique App Operator. Fortunately, when
+the `kubectl-gs` is used then both of these fields are set correctly when templating an app, when the
+`--in-cluster` flag is used, see [kubectl gs template app]({{< relref "/ui-api/kubectl-gs/template-app.md" >}}),
+so that there is no need to remember about it.
+
+<div class="feedback well">
+
+<h5><i class="fa fa-help-outline"></i> Can I use it to install any app in the Management Cluster?</h5> No, but it
+will work for some apps. The Management Cluster's Chart Operator uses a limited set of permissions when
+installing user-requested apps, making it prohibit for him to crate certain cluster-scoped resources, like
+"ClusterRole", or to access certain namespaces, like "kube-system". When the app being requested is limited to
+the namespace you have access to, and does not go beyond, then installing it within the Management Cluster
+should work.
+
+</div>
+
+#### Configuring target Workload Cluster for the nested App CRs
+
+Once the App CR name and configuration for unique App Operator are taken care of, the only thing left is to
+configure the bundle so that nested App CRs are placed into correct Workload Cluster namespace, and with the
+correct configuration. It requires two steps.
+
+Firstly, the App CR's `.metadata.namespace` and `.spec.namespace` fields must be equal, meaning must reference
+the same namespace. Let's consider the snippet from the previous paragraph and enrich it with these settings.
+
+```yaml
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+  name: bundle-wc1
+  namespace: wc1
+spec:
+  ...
+  kubeConfig:
+    inCluster: true
+  name: bundle-app
+  namespace: wc1
+  ...
+```
+
+Secondly, each app bundle Helm Chart exposes the `clusterName` and `organization` configuration options in its
+`values.yaml` file, see below.
+
+```yaml
+clusterName: ""
```

Comment:
My only question is: Why did we move to clusterName instead of clusterID ?

Reply:
I don't remember of when exactly this decision has been taken, but I doubt it was one team behind it as there seem to be a global movement towards cluster names. At least I've seen it being used all over the places.